PLBenchmarks/edges.py

```diff
@@ -0,0 +1,174 @@
+"""
+edges.py
+Functions and classes for handling the perturbation edges.
+"""
+import os
+import pandas as pd
+import numpy as np
+import yaml
+
+import PLBenchmarks
+import PLBenchmarks.ligands
+import PLBenchmarks.targets
```

Comment:
```suggestion
import .ligands
import .targets
```
1. Likely don't want to be importing the whole module into itself
1. Can use relative imports for the modules

Reply:
`import .ligands` does not work. Changed it to `from . import ligands, targets` in commit https://github.com/openforcefield/PLBenchmarks/pull/3/commits/cd8d5c0124b4483323b2d9c96df31755602cadd8